packages/cosmic-swingset/src/types.d.ts

```diff
@@ -0,0 +1,43 @@
+import { QueuedActionType } from '@agoric/internal/src/action-types.js';
+import type {
+  CoreEval,
+  CoreEvalSDKType,
+} from '@agoric/cosmic-proto/agoric/swingset/swingset.js';
+
+interface BlockInfo {
+  blockHeight: number;
+  blockTime: number;
+}
```

Comment:
Remember that Golang `int64` is JSON `string`:

```suggestion
interface ActionContext<T extends string> {
  type: T;
  blockHeight: string;
  blockTime: string;
}
```